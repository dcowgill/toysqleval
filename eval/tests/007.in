create table test (
    n integer,
    s varchar,
    f number,
    t timestamp,
    b boolean
);

insert into test values (42, 'hello', 3.14, 'Sun May 10 21:39:19 -0400 2009', true);
insert into test values (99, 'world', 0.05, 'Sat May 09 21:39:19 -0400 2009', false);
insert into test values (17, 'blarg', 9.51, 'Mon May 11 21:39:19 -0400 2009', true);

select n from test where s = 'blarg';
select n from test where n > 17 and n != 42;
select n from test where s = 'hello' or b = false;
select n from test where t = '2009-05-09T21:39:19-04:00';
select n from test where t = '2009-05-09T21:39:19-04:00' and s != 'world';

update test set n = n*2, f = f+1, s = s||s
 where s != 'world';

select * from test;
