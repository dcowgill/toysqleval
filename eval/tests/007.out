OK
OK
OK
OK
 n 
----
 17
 n 
----
 99
 n 
----
 42
 99
 n 
----
 99
 n
---
OK
 n  | s            | f                 | t                         | b    
----+--------------+-------------------+---------------------------+-------
 84 | "hellohello" | 4.140000000000001 | 2009-05-10T21:39:19-04:00 | true 
 99 | "world"      | 0.05              | 2009-05-09T21:39:19-04:00 | false
 34 | "blargblarg" | 10.51             | 2009-05-11T21:39:19-04:00 | true 
